src/main/java/graphql/execution/ExecutionContext.java

```diff
@@ -368,6 +369,16 @@ public ResultNodesInfo getResultNodesInfo() {
         return resultNodesInfo;
     }
 
+    private ExecutableNormalizedOperation createExecutableNormalizedOperation() {
+        var parameters = new InstrumentationParsedNormalizedOperation(executionInput, graphQLSchema);
+        var instrument = instrumentation.beginParsedNormalizedOperation(parameters, instrumentationState);
+        var result = ExecutableNormalizedOperationFactory.createExecutableNormalizedOperation(graphQLSchema, operationDefinition, fragmentsByName, coercedVariables);
+        if (instrument != null) {
+            instrument.onCompleted(result, null);
+        }
+        return result;
+    }
+
```

Comment:
naming is hard

but is it `beginParsedNormalizedOperation`

I am not sure its parsed - I think `beginCreateNormalizedOperation` is a better name

Reply:
and the parameters need to reflect this name